external-crates/move/crates/move-compiler/src/command_line/compiler.rs

```diff
@@ -756,46 +785,26 @@ fn generate_interface_files_for_deps(
     Ok(())
 }
 
+// TODO this should really be done by the package system, with the interface files stuffed into the
+// build/ directory for the package. This would give a more consistent location for errors.
```

Comment:
Speaking of pre-package system relics... it would be really great to make this a package system feature someday